SoftLayer/CLI/user/remove_access.py

```diff
@@ -0,0 +1,38 @@
+"""User remove access to devices."""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('identifier')
+@click.option('--hardware', '-h',
+              help="Display hardware this user has access to.")
+@click.option('--virtual', '-v',
+              help="Display virtual guests this user has access to.")
+@click.option('--dedicated', '-l',
+              help="dedicated host ID ")
+@environment.pass_env
+def cli(env, identifier, hardware, virtual, dedicated):
+    """User details."""
+
+    mgr = SoftLayer.UserManager(env.client)
+    device = ''
+    result = False
+    if hardware:
```

Comment:
Before the conditions, it would be good to add that at least one flag must be added.
![image](https://user-images.githubusercontent.com/43519769/182639691-4b6aa95f-7de7-4540-9019-55a973af6889.png)